tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -652,6 +652,8 @@ class CompletionArgSuite extends BaseCompletionSuite {
        |""".stripMargin,
     """|foo = : Int
        |foo = a : Int
+       |foo: Int
+       |fooBar: Int
```

Comment:
I think https://github.com/scalameta/metals/pull/5324 should fix those additional results.